Test/New-WhiskeyProGetUniversalPackage.Tests.ps1

```diff
@@ -866,6 +883,17 @@ Describe 'New-WhiskeyProGetUniversalPackage.when cleaning' {
     Then7zipShouldNotExist
 }
 
+Describe 'New-WhiskeyProGetUniversalPackage.when initializing' {
+    $file = 'project.json'    
+    Given7ZipIsInstalled

```

Comment:
Don't install 7zip as part of this test since it is testing that 7zip gets installed.

Reply:
Done